WORKSPACE

```diff
@@ -16,6 +16,13 @@ http_archive(
     ],
 )
 
+http_archive(
```

Comment:
This should live in protobuf_deps.bzl under a maybe.

Reply:
I don't think it should.  This dependency is brought in by googletest, which we don't propagate to clients via protobuf_deps.  Note: *this* one is somewhat unrelated, I hit errors that were likely pre-existing due to this missing dependency.  I suspect the root cause is from when we upgraded googletest